src/platform/src/Capability.php

```diff
@@ -33,4 +33,7 @@ class Capability
 
     // FUNCTIONALITY
     public const TOOL_CALLING = 'tool-calling';
+
+    // METADATA
+    public const METADATA_USAGE = 'metadata-usage';
```

Comment:
what's your reasoning/motivation to add it as capability?

at least it's not used at this point, not sure about it and wonder if that's really a model capability.
I'd say we remove this for now - if you don't have something concrete in mind